src/components/SettingsPane/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+	Box,
+	Card,
+	CardContent,
+	CardHeader,
+	Stack,
+	Typography,
+} from '@mui/material';
+import { color } from '@balena/design-tokens';
+
+export interface SettingsPaneProps {
+	title?: string | React.ReactNode;
+	children: React.ReactNode;
+	danger?: boolean;
+}
+
+export interface SettingsPaneItemProps {
+	title: string | React.ReactNode;
+	description?: string | React.ReactNode;
+	action?: React.ReactNode;
+}
+
+export const SettingsPane = ({
```

Comment:
I don't think `Pane` is the right word. When I think "Pane" I think of something that expands, i.e. a sliding door (slide open a sidebar by clicking a collapsed version of it)

Perhaps `Settings` is enough? Or `SettingsCard`?

Reply:
`SettingsCard` 👌